src/Morphic-Widgets-Scrolling/Slider.class.st

```diff
@@ -110,19 +122,20 @@ Slider >> initializeSlider [
 Slider >> mouseDownInSlider: event [ 
 
 	slider borderStyle style == #raised
-		ifTrue: [slider borderColor: #inset].
+		ifTrue: [slider borderColor: #inset. self dragging: true].
```

Comment:
I don't understand why this depends on the border style. Could you add a comment on this?